docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -0,0 +1,209 @@
+# Configuration Vocabulary
+Configuration Vocabulary lists all available configurations provided by `application.yml`.
+
+Module | Provider | Settings | Value(s) and Explanation | System Environment Variable¹ | Default |
+----------- | ---------- | --------- | --------- |--------- |--------- |
+core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode OAP open the service to the agents, analysis and aggregate the results and forward the results for distributed aggregation. Aggregator mode OAP receives data from Mixer and Receiver role OAP nodes, and do 2nd level aggregation. **Mixer** means being Receiver and Aggregator both.|SW_CORE_ROLE|Mixed|
+ - | - | restHost| Binding IP of restful service. Services include GraphQL query and HTTP data report|SW_CORE_REST_HOST|0.0.0.0|
```

Comment:
The table format breaks from here , because of the leading `-`

Reply:
Fixed. Local and GitHub markdown seems to use different engines.